packages/viewer/src/lib/scenes/scene.ts

```diff
@@ -587,6 +587,11 @@ export class SceneItemOperationsBuilder
 }
 
 /**
+ * @ignore
+ * @beta PMI Annotation operations are being actively developed. These APIs
+ * should be considered unstable, and will potentially change with following
+ * releases.
```

Comment:
Ignoring + adding a beta tag to all of the PMI Annotation operations so it's clear they're not ready for broader usage yet. Not all of the operations will work today, and we don't provide a formal element for interacting with ModelViews and PMI Annotations